lib/perl/Genome/InstrumentData/Command/Import/Basic.pm

```diff
@@ -104,6 +98,7 @@ sub execute {
     my $inputs = $self->_gather_inputs_for_workflow;
     return if not $inputs;
 
+    $DB::single=1;
```

Comment:
Hark! A breakpoint lurks in the shadows, ready to leap out at unsuspecting debuggers by.